### PR TITLE
Update ppc64le cuda-vector-add 2.0 to be based on Cuda10 base image.

### PR DESCRIPTION
**Pull request by jiayingz**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/images/cuda-vector-add/README.md [hunk 1]
```diff
@@ -15,4 +15,4 @@ $ make push
 ## Version history:
 
 1.0: build cuda-vector-add from CUDA 8.0.
-2.0: build amd64 cuda-vector-add from CUDA 10.0 and ppc64le from CUDA 9.2.
+2.0: build cuda-vector-add from CUDA 10.0
```
Comment (ixdy) [inline test/images/cuda-vector-add/README.md]: should this be a 2.1?

Comment (jiayingz) [reply test/images/cuda-vector-add/README.md]: I think we can stay with 2.0. This change is to address the comment https://github.com/kubernetes/kubernetes/pull/73823#discussion_r254953524 that unfortunately wasn't caught before the submit.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**